keras_cv/layers/preprocessing/maybe_apply.py

```diff
@@ -29,6 +29,8 @@ class MaybeApply(BaseImageAugmentationLayer):
         rate: controls the frequency of applying the layer. 1.0 means all elements in
             a batch will be modified. 0.0 means no elements will be modified.
             Defaults to 0.5.
+        batchwise: (Optional) bool, whether or not to pass entire batches to the
```

Comment:
Is the expected behavior here that if batchwise=true, we decide on a per-batch level whether to apply a layer?

If so, might be worth documenting that.

Reply:
Updated the docstring, updated the behavior, etc